src/components/SelectLang.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <q-btn-dropdown flat label="Language">
+  <q-btn-dropdown flat :label="$t('language')">
```

Comment:
```suggestion
  <!-- Don't translate the word "Language" since it will probably be better understood by the user seeking for this option rather than the translated word to a language that the user wants to change. -->
  <q-btn-dropdown flat :label="Language">
```

Reply:
Això estava fet expressament. No poso la mà al foc per aquesta idea, però sí que crec que té la seva lògica. Faltava clarament el comentari, però!